apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/test/java/gov/cms/bfd/pipeline/rda/grpc/RdaLoadJobIT.java

```diff
@@ -125,6 +148,26 @@ public void invalidFissClaimTest() throws Exception {
         });
   }
 
+  /**
+   * All of our test claims should be valid for our IT tests to succeed. This test ensures this is
+   * the case and catches any incompatibility issues when a new RDA API version contains breaking
+   * changes.
+   */
+  @Test
+  public void mcsClaimsAreValid() throws Exception {
+    final ImmutableList<McsClaimChange> expectedClaims =
```

Comment:
I've noticed a number of variables declared as `ImmutableList` (or similar data structure) in the code in various places I believe.  Is there a specific need for this?  Typically you'd want to declare the interface `List` so that the code isn't quite as restricting.  For instance, later we might want to update all this logic to use JDK11 features like `List.of()` instead of `ImmutableList.of()` if it seemed appropriate.

Reply:
I agree but ImmutableList has declarative value since it makes it perfectly clear that this is an immutable field.